core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -196,9 +196,10 @@ else if (element instanceof LikeClause) {
                 TableHandle likeTable = redirection.getTableHandle()
                         .orElseThrow(() -> semanticException(TABLE_NOT_FOUND, statement, "LIKE table '%s' does not exist", originalLikeTableName));
 
+                LikeClause.PropertiesOption propertiesOption = likeClause.getPropertiesOption().orElse(EXCLUDING);
                 QualifiedObjectName likeTableName = redirection.getRedirectedTableName().orElse(originalLikeTableName);
-                if (!tableName.getCatalogName().equals(likeTableName.getCatalogName())) {
-                    String message = "CREATE TABLE LIKE across catalogs is not supported";
+                if (propertiesOption == INCLUDING && !tableName.getCatalogName().equals(likeTableName.getCatalogName())) {
+                    String message = "CREATE TABLE LIKE table INCLUDING PROPERTIES across catalogs is not supported";
```

Comment:
Since `INCLUDING PROPERTIES` is implicit, maybe I should reword this to something like `CREATE TABLE LIKE table across catalogs is only supported with EXCLUDING PROPERTIES`?